docs/code/readme_standards.md

```diff
@@ -108,20 +143,18 @@ All tables should follow this pattern:
 ```
 <COLUMN_NAME> | Description
 --- | ---
-<INLINE_CODE> | <ONE_SENTENCE_DESCRIPTION>
+<INLINE_CODE> | <ONE_LINE_DESCRIPTION>
 ```
 
-* `COLUMN_NAME` should be title case
-* `INLINE_CODE` should always be formatted as inline code
-* `ONE_SENTENCE_DESCRIPTION` is one sentence
+* `COLUMN_NAME` should be title case, e.g. Method Signature
+* `INLINE_CODE` should always be formatted as inline code, e.g. `foo`
+* `ONE_LINE_DESCRIPTION` should not be more than one line long
+
+If you need more than one sentence to describe code, create a row with a
+summary description. Then create a note below the table with the longer description, formatted in this way:
 
-If you need more than one sentence to describe code, then create a row with a
-summary description. Then create another subsection below the table, titled
-with `INLINE_CODE`, and write the longer description in this section. Do not
-add multiple sentences to the tabular format, as this will make the table more
-difficult to read.
+~~~
+>_NOTE_: This is the long description.
+~~~
 
-If a table feels too long, and there is a pattern to the rows, you can use an
-uppercase angled brackets, e.g. `<FOO>` format to explain the pattern. Add a
-note above the table detailing what the uppercase angled brackets means,
-including an example of a valid pattern match.
+Do not add multiple sentences to the tabular format, as this will make the table more difficult to read.
```

Comment:
I don't 100% agree with this...

Firstly, "Mandatory." or "Optional." is sometimes the first sentence :) (though for that matter maybe we should standardize on only pointing out mandatory and implying everything else is optional, 'cause there tend to be far more optional things than mandatory ones)

Secondly, we sometimes add a second sentence to explain the meaning of parameters (especially optional ones), and I'm not sure having separate NOTEs for a bunch of mixins would scale for that purpose (unless multiple mixins follow a similar pattern).

Reply:
Yeah good point about Mandatory/Optional. Changed "sentence" to "line", hopefully that gets the idea across.
The idea here is to keep the table so users can skim it to find what they're looking for, and then go into details later. It doesn't always fit on one line but that's the goal, and authors can use their own discretion. The Ripple README is an example of separating details out into notes.